src/store/inventory-actions.js

```diff
@@ -193,6 +194,11 @@ export const deleteEntity = (systems, displayName) => ({
     }
 });
 
+export const setRosTabVisibility = (shouldShowRosTab) => ({
```

Comment:
I don't think you need this action, it should be enough to wait for the `ACTION_TYPES.LOAD_SYSTEM_PROFILE`. If such action is not fired when user is on `ROS` tab by default, we should fire the `systemProfile` action directly when `InventoryDetail` route is rendered.